### ISSUE TITLE
Difference between KC_POWER and KC_PWR

### ISSUE THREAD

**Issue by Dbroqua** [qmk/qmk_firmware#1994]
Hello,

what is the difference between KC_PWR and KC_POWER?

Regards

**Comment by skullydazed** [comment]
I looked into this and found a document from Microsoft describing the difference between the two. It seems that that `KC_POWER` simply sends scancode 0x66 in the Keyboard/Keypad Page 0x07. `KC_PWR` implements the newer System Control interface, which is the recommended method.

You can fetch the whole document from <https://msdn.microsoft.com/en-US/library/windows/hardware/dn550976>, it's called Archive: Key Support, Keyboard Scan Codes, and Windows: Keyboard Scan Code Specification. I've attached an image of the relevant section. 

TL;DR: Use `KC_PWR`.

![screen shot 2017-11-10 at 12 40 03 pm](https://user-images.githubusercontent.com/9553982/32677806-c3b16ef8-c614-11e7-9f21-f6ebc7743280.png)

**Comment by Dbroqua** [comment]
Thanks!
